Jenkinsfile

```diff
@@ -0,0 +1,199 @@
+pipeline_name="vivarium_gates_nutrition_optimization"
+conda_env_name="${pipeline_name}-${BUILD_NUMBER}"
+conda_env_path="/tmp/${conda_env_name}"
+// defaults for conda and pip are a local directory /svc-simsci for improved speed.
+// In the past, we used /ihme/code/* on the NFS (which is slower)
+shared_path="/svc-simsci"
+
+pipeline {
+  // This agent runs as svc-simsci on node simsci-slurm-sbuild-p01.
+  // It has access to standard IHME filesystems and singularity
+  agent { label "svc-simsci" }
+
+  options {
+    // Keep 100 old builds.
+    buildDiscarder logRotator(numToKeepStr: "100")
+
+    // Wait 60 seconds before starting the build.
+    // If another commit enters the build queue in this time, the first build will be discarded.
+    quietPeriod(60)
+
+    // Fail immediately if any part of a parallel stage fails
+    parallelsAlwaysFailFast()
+  }
+
+  parameters {
+    string(
+      name: "SLACK_TO",
+      defaultValue: "simsci-ci-status",
```

Comment:
Im still not sure we would want to be pinged on slack every time a model repo fails but it depends what the build is for. I think for pull requests/cron jobs, that is something we would want but not on pushes.

Reply:
My opinion is we really _must_ be notified one way or another. Once we sort out how to have these builds trigger on push/pull request and then appropriate merge restrictions then we can stop notifying on slack